### ISSUE TITLE
FAIL compiling github.com/cpmech/gosl/la/oblas

### ISSUE THREAD

**Issue by YongmingFrankGe** [cpmech/gosl#61]
Hi when I followed the README, installed the dependencies, and then ran `./all.bash`
I got the following failure:
```
=== compiling la/oblas =============================================================
   . . . testing . . .   SplitJoinComplex
   . . . testing . . .   Conversions01. real
   . . . testing . . .   Conversions02. complex
   . . . testing . . .   Matrix01. real
   . . . testing . . .   Matrix02. complex
   . . . testing . . .   Ddot01
   . . . testing . . .   Dscal01
   . . . testing . . .   Daxpy01
   . . . testing . . .   Zaxpy01
   . . . testing . . .   Dgemv01
   . . . testing . . .   Zgemv01
   . . . testing . . .   Dger01: a = 100 + 0.5⋅u⋅vᵀ
   . . . testing . . .   Dgemm01: 0.5⋅a⋅b + 2⋅c
   . . . testing . . .   Dgemm02: 0.5⋅a⋅bᵀ + 2⋅c
   . . . testing . . .   Dgemm03: 0.5⋅aᵀ⋅b + 2⋅c
   . . . testing . . .   Dgemm04: 0.5⋅aᵀ⋅bᵀ + 2⋅c
   . . . testing . . .   Zgemm01
   . . . testing . . .   Dgesv01
   . . . testing . . .   Zgesv01. low accuracy
   . . . testing . . .   Dgesvd01
--- FAIL: TestDgesvd01 (0.00s)
    print.go:39: u [2][3]  1 != -1 |diff| = 2
    print.go:39: vt [2][1]  -1 != 1 |diff| = 2
   . . . testing . . .   Dgesvd02
   . . . testing . . .   Dgesvd03
   . . . testing . . .   Zgesvd01
   . . . testing . . .   Zgesvd02
   . . . testing . . .   Dgetrf01. Dgetrf and Dgetri
   . . . testing . . .   Zgetrf01. Zgetrf and Zgetri
   . . . testing . . .   Dsyrk01
   . . . testing . . .   Dsyrk02
   . . . testing . . .   Zsyrk01
   . . . testing . . .   Zherk01
   . . . testing . . .   Dpotrf01
   . . . testing . . .   Zpotrf01
   . . . testing . . .   Dgeev01
FAIL
exit status 1
FAIL    github.com/cpmech/gosl/la/oblas 0.014s
``
Seems only 1 test is failing, I'm wondering what does it mean by this test failing and is there a solution to it? Please let me know if you need more information.

**Comment by cpmech** [comment]
Hi, thanks for your message.

Depending on the versions of OpenBLAS, the vectors from the SVD decomposition may appear on different columns. However, the results are still correct.

I have disabled the direct comparison of vectors in those tests. However, I still have another layer of test for the SVD regardless of the "fixed" vectors that I have provided. All should be working now.

Can you please try again with the latest code?

Cheers.

**Comment by YongmingFrankGe** [comment]
Great Thanks! I'm ablt to test and install.
However, I just realized that the reason why I need to install this dev tool is because I got the following when I tried to do something else:
```
 no required module provides package github.com/cpmech/gosl/graph; to add it:
        go get github.com/cpmech/gosl/graph
```
I don't see /gosl/graph package under the main repo. Do we still have it?

**Comment by cpmech** [comment]
Oh, this is due to different versions. The version with `graph` is the old one https://github.com/cpmech/gosl/tree/stable-1.1.3 Please try to use the old version and see if it works

**Comment by YongmingFrankGe** [comment]
Thanks. I'm trying the old version and encountered the same initial problem
```
--- FAIL: TestDgesvd01 (0.01s)
    print.go:39: u [2][3]  1 != -1 |diff| = 2
    print.go:39: vt [2][1]  -1 != 1 |diff| = 2
```
Would you mind update the stable-1.1.3 branch as well?

**Comment by cpmech** [comment]
All right. I've fixed the stable-1.1.3 branch as well. Please try and let me know. Thanks!

**Comment by YongmingFrankGe** [comment]
I've tried and it works. Thank you very much! Feel free to close the ticket.
